src/controller-api/models/motion-data.interface.ts

```diff
@@ -32,7 +32,7 @@ export interface MotionDataWithTimestamp extends MotionData {
 /**
  * Motion data with quaternion.
  */
-export interface MotionDataWithQuaternion extends MotionData {
+export interface MotionDataWithQuaternion extends MotionDataWithTimestamp {
```

Comment:
This should remain as MotionData only as steam controller does not have a timestamp.